themes/src/main/resources-community/theme/base/account/messages/messages_es.properties

```diff
@@ -133,6 +187,9 @@ federatedIdentityLinkNotActiveMessage=Esta identidad ya no est\u00E1 activa
 federatedIdentityRemovingLastProviderMessage=No puedes eliminar la \u00FAltima identidad federada porque no tienes fijada una contrase\u00F1a.
 identityProviderRedirectErrorMessage=Error en la redirecci\u00F3n al proveedor de identidad
 identityProviderRemovedMessage=Proveedor de identidad borrado correctamente.
+identityProviderAlreadyLinkedMessage=La identidad devuelta por el proveedor de identidad ya est\u00E1 asociada a otro usuario.
```

Comment:
Here I think it should use the parameter slot {0} as in the English version:
"La identidad devuelta por {0} ya est\u00E1 asociada a otro usuario."